### DISCUSSION TITLE
Q: 使用github pages会出现CORS跨域访问的问题，有没有不绑定域名的解决方法吗 || Q: There will be CORS cross-domain access problems when using github pages. Is there a solution without binding a domain name?

### DISCUSSION THREAD

**Discussion by ganyuo** [walinejs/waline#2852] [Q&A]
### Describe the bug | 问题描述

我按照waline的[快速上手](https://waline.js.org/guide/get-started/)文档在github pages上部署好了，在vercel的[demo](https://waline-ao2gukw9r-ganyuooutlookcoms-projects.vercel.app/)上是可以正常评论的，但是在github pages上打开会报CORS跨域访问问题，下面是chrome浏览器的报错信息。请问除了绑定自定义域名外，是否有其他解决方法？
> Access to fetch at 'https://waline-ao2gukw9r-ganyuooutlookcoms-projects.vercel.app/api/comment?path=%2Flearn_note%2Fqt_learn%2F&pageSize=10&page=1&lang=zh-CN&sortBy=insertedAt_desc' from origin 'https://ganyuo.github.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

### Website URL | 问题网站

https://ganyuo.github.io/learn_note/qt_learn/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

I deployed it on github pages according to waline's [Quick Start](https://waline.js.org/guide/get-started/) document, and in vercel's [demo](https://waline-ao2gukw9r- Comments can be made normally on ganyuooutlookcoms-projects.vercel.app/), but when opened on github pages, a CORS cross-domain access problem will be reported. The following is the error message of the Chrome browser. In addition to binding a custom domain name, are there any other solutions?
> Access to fetch at 'https://waline-ao2gukw9r-ganyuooutlookcoms-projects.vercel.app/api/comment?path=%2Flearn_note%2Fqt_learn%2F&pageSize=10&page=1&lang=zh-CN&sortBy=insertedAt_desc' from origin 'https: //ganyuo.github.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

### Website URL | Problem website

https://ganyuo.github.io/learn_note/qt_learn/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
serverURL 填错了，你现在填的是 vercel 某次部署的预览地址，需要填项目的固定地址，自己在首页找找吧。

---
The serverURL is filled in incorrectly. What you are filling in now is the preview address of a certain vercel deployment. You need to fill in the fixed address of the project. Find it on the homepage.

**Reply by ganyuo** [reply]
修改serverURL后正常了，非常感谢！

---
After modifying the serverURL, it became normal. Thank you very much!
